src/style/content_style.rs

```diff
@@ -51,7 +51,7 @@ impl ContentStyle {
     /// You can add more attributes by calling this method multiple times.
     #[inline]
     pub fn attribute(mut self, attr: Attribute) -> ContentStyle {
```

Comment:
Would it be an idea, to create a method that accepts `Attributes` as well?

Reply:
probably. Calling extend.

Reply:
Oke